### ISSUE TITLE
A11y Audit Reporting

### ISSUE THREAD

**Issue by seanforyou23** [patternfly/patternfly-next#522]
Now that we've settled on a tooling framework for auditing and testing our accessibility compliance efforts (https://github.com/patternfly/patternfly-next/issues/466), we need a mechanism in place for exposing and storing the reports we generate.

Done criteria:
- Results posted back as part of PR status, for example affecting reports in screenshot below

![image](https://user-images.githubusercontent.com/5942899/43649765-01e5084c-970c-11e8-8f8c-0ba97483c6b9.png)

Requirements:
- Contributors can access audit results
- Failing an audit does not prevent merging of a PR (for now anyway)
- Selecting "Details" link from a failed build navigates a user to the travis build log where they can find more information about the nature of the violation(s)

Nice to have:
- Nicely formatted comment posted back to PR with specific details of found violations
